blog/_src/posts/2017-09-27-tutorial-contributing-to-racket.md

```diff
@@ -0,0 +1,414 @@
+    Title: Tutorial: Contributing to Racket
+    Date: 2017-09-27T01:07:20
+    Tags: 
+
+*posted by Ben Greenman*
+
+Racket is on GitHub and contributions are always welcome.
+In the spirit of jolly cooperation, this post describes how to submit a quality pull request to a Racket package.
+
+<!-- more -->
+
+What follows is a collection of advice and best-practices for contributing to a
+ Racket package (as of Fall 2017).
+The broad outline for "how to contribute" is the same for Racket as for any
+ mid-sized software project, but there are some important details.
+The real purpose of this post is to list those details in one easy-to-find place.
+
+> See also: [Contributing to Racket](http://blog.racket-lang.org/2012/11/tutorial-contributing-to-racket.html) (the 2012 edition)
+> and [Guide for Infrequent Contributors](http://www.greghendershott.com/2013/04/a-guide-for-infrequent-contributors-to-racket.html)
+
+## Recipe for a Pull Request
```

Comment:
In the interests of avoiding wastes of contributor time, something might be said about what kind of PRs are worth doing / likely to be merged. For instance, corrections and updates to the documentation are easy, small, and self-contained, and thus tend to be approved quickly. Whereas a reimplementation of `cons` is going to attract more scrutiny.